packages/api-client/src/api/urlResolver/index.ts

```diff
@@ -10,6 +10,7 @@ import type { Context } from '../../types/context';
  * @param context VSF Context
  * @param url the URL to be resolved
  * @param [customQuery] (optional) - custom GraphQL query that extends the default one
+ * @deprecated - use route query instead
```

Comment:
https://github.com/vuestorefront/magento2/pull/1078/files#r884967163
Regarding the linked comment, I think it'd be easier to actually point to what the user is supposed to use. If I read this comment i wouldn't know what to use instead.

whereas
```suggestion
 * @deprecated - use getRoute instead
```
is more exact

Reply:
ok, thanks!